python/orca/src/bigdl/orca/data/file.py

```diff
@@ -385,7 +391,7 @@ def put_local_file_to_remote(local_path, remote_path, filemode=None):
                 process = subprocess.Popen(chmod_cmd, shell=True)
```

Comment:
Why the functions between Line 217 and 391 are not typed?

Reply:
> Why the functions between Line 217 and 391 are not typed?

Not sure if these are needed, add now